packages/exchange/src/pods/order/product.dto.ts

```diff
@@ -38,6 +38,9 @@ export class ProductDTO {
     @Validate(GridOperatorValidator)
     public gridOperator?: string[];
 
+    @IsOptional()
+    public externalDeviceId?: { id: string; type: string };
```

Comment:
Replace with the proper type `IExternalDeviceId`.

https://github.com/energywebfoundation/origin/blob/52e5fccdeb303133b3db9413fc317e4a1c60ebca/packages/origin-backend-core/src/Device.ts#L10